packages/cdktf-cli/test/lib/index.test.ts

```diff
@@ -116,10 +116,9 @@ describe("CdktfProject", () => {
         },
       });
 
-      return expect(cdktfProject.diff()).rejects.toMatchInlineSnapshot(`
-                "terraform init errored with: 
-                Error: Usage Error: Found more than one stack, please specify a target stack. Run cdktf diff <stack> with one of these stacks: first, second "
-              `);
+      return expect(cdktfProject.diff()).rejects.toMatchInlineSnapshot(
+        `[Error: Error: Usage Error: Found more than one stack, please specify a target stack. Run cdktf diff <stack> with one of these stacks: first, second ]`
```

Comment:
Is `Error: Error: Usage Error:` shown to users? 😅

Reply:
No, `Error: Usage Error:` is shown